versions/1/data_stream/manifest.spec.yml

```diff
@@ -68,32 +68,32 @@ spec:
           - name
           - type
   properties:
-    name:
-      description: Name of dataset
+    data_stream:
```

Comment:
@ycombinator Is this rename correct? `name->data_stream`?

Reply:
Yes, see https://github.com/elastic/package-spec/pull/24#discussion_r470464135.

Reply:
Ah... and I even approved it :) Ok, let's leave it here.

Reply:
shouldn't this be `name`->`dataset`, not `data_stream`? I see it was `dataset` up until the last commit f706b65

Reply:
Hmm, yes, you're right — it should be `dataset` not `data_stream` per https://github.com/elastic/package-spec/pull/24#discussion_r470464135. Fixing...

Reply:
Fixed in 37fd8917ea45530bf5885e50efa0c76344d0e0a0. Thanks!